lib/thermos/beverage.rb

```diff
@@ -2,15 +2,16 @@
 
 module Thermos
   class Beverage
-    attr_reader :key, :model, :deps, :action, :lookup_key, :filter
+    attr_reader :key, :model, :deps, :action, :lookup_key, :filter, :queue
 
-    def initialize(key:, model:, deps:, action:, lookup_key: nil, filter: nil)
+    def initialize(key:, model:, deps:, action:, lookup_key: nil, filter: nil, queue: nil)
       @key = key
       @model = model
       @lookup_key = lookup_key || :id
       @filter = filter || nil
       @deps = generate_deps(model, deps)
       @action = action
+      @queue = queue || 'default'
```

Comment:
maybe there is a constant provided by rails so this is not a magic string

Reply:
We could use `ActiveJob::Base.default_queue_name`, which also respects overrides in the configs using `config.active_job.default_queue_name = 'custom_default'`.